components/Cardforcasestudies.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Link from 'next/link';
+import TextTruncate from 'react-text-truncate';
+interface CardProps {
+  title: string;
+  body: string;
+  icon?: string;
+  link?: string;
+  image?: string;
+}
+
+const CardBody = ({ title, body, icon, link, image }: CardProps) => {
+  return (
+    <div className='group relative h-full w-full max-w-lg rounded-lg border border-gray-200 bg-white p-6 px-12 shadow-3xl transition-colors delay-[150ms] ease-in-out hover:bg-slate-100'>
+      <div className='flex justify-center '>
+        {image && <img src={image} className='h-32 w-36 p-2' />}
+      </div>
+      <div className='flex flex-row items-start mb-6'>
+        {icon && (
+          <span className='mr-6 flex h-14 w-14 flex-shrink-0 items-center justify-center rounded-lg border bg-blue-200 px-3 text-gray-900'>
+            <img src={icon} alt={title} className='h-full w-full' />
+          </span>
+        )}
+        <p className='mt-1 items-center text-[0.9rem] font-bold text-gray-900'>
```

Comment:
Hey there! Great work on the code! I noticed the header seemed a bit small. Might we consider text-h5 or text-h6 for better readability?
```suggestion
        <p className='mt-1 items-center text-h5 font-bold text-gray-900'>
```

Reply:
Hey! Thanks for the feedback .I will work on it. Thanks:)